ingestion.py

```diff
@@ -0,0 +1,261 @@
+import matplotlib.pyplot as plt
+import numpy as np
+import scipy.io as sio
+import os
+from datetime import datetime
+import re
+
+from pipeline.helper_functions import get_one_from_nested_array, extract_datetime
+
+############## Dataset #################
+# (this needs to be placed in a config file somewhere)
+data_dir = os.path.abspath('..//NWB_Janelia_datasets//crcns_ssc5_data_HiresGutnisky2015//')
+meta_data_dir = os.path.join(data_dir,'metadata')
+sess_data_dir = os.path.join(data_dir,'datafiles')
+
+#################################
+import datajoint as dj
+
+dj.config['database.host'] = 'tutorial-db.datajoint.io' # this goes to config file as well
```

Comment:
do not specify the host in the code

Reply:
@ttngu207 I had a same comment on this line in the previous PR. Please check against my comments (sorry, I know there are a lot), and ensure you have addressed all of them. Until then, mark this PR as `[WIP]`